### PR TITLE
Feature/71110

### PR DESCRIPTION
**Pull request by ofrankowska**
[71110](https://lab.snowdog.pro/bosca/theme-frontend-bosca/-/merge_requests/29)

### CODE REVIEW

File: Snowdog_Components/components/Templates/product-view/_product-view.scss [hunk 1]
```diff
@@ -153,7 +153,7 @@
         display: $product-view__price-display;
 
         @include mq($product-view__price-display-bq) {
```
Comment (anqaka) [inline Snowdog_Components/components/Templates/product-view/_product-view.scss:155]: is not a typo? shouldn't be `$product-view__price-display-mq` instead?

File: Snowdog_Components/components/Molecules/tab/_tab-variables.scss [hunk 2]
```diff
@@ -1,33 +1,42 @@
...
-$tab__content-width               : 100%;
-$tab__content-line-height         : 32px !default;
+$tab__content-z-index              : $z-index-lowest !default;
+$tab__content-z-index\@medium      : $z-index-low !default;
+$tab__content-padding              : 64px 0 !default;
+$tab__content-padding\@medium      : 72px 104px !default;
+$tab__content-padding\@large       : 72px 80px !default;
+$tab__content-padding\@extra-large : 72px 112px !default;
+$tab__content-padding--secondary   : 64px 0 !default;
+$tab__content-width                : 100%;
```
Comment (anqaka) [inline Snowdog_Components/components/Molecules/tab/_tab-variables.scss:32]: missing `!default`

File: Snowdog_Components/components/Templates/product-view/_product-view-variables.scss [hunk 3]
```diff
@@ -60,6 +60,7 @@ $product-view__rating-border-width                          : 0 0 $border-width-
 
 // price
 $product-view__price-display                                : none !default;
+$product-view__price-display\@display-bq                    : block !default;
```
Comment (anqaka) [inline Snowdog_Components/components/Templates/product-view/_product-view-variables.scss:63]: shouldn't be `mq`?

File: Snowdog_Components/components/Templates/product-view/_product-view.scss (hunk 1 above)
Comment (ofrankowska) [reply Snowdog_Components/components/Templates/product-view/_product-view.scss:155]: yeah, that would make sense. I was confused what bq is supposed to mean xd

Comment (anqaka) [reply Snowdog_Components/components/Templates/product-view/_product-view.scss]: XD

**Review by anqaka — APPROVED** [review]

**Pull request merged**